src/Umbraco.Web.UI.Client/src/common/services/formhelper.service.js

```diff
@@ -89,9 +89,9 @@ function formHelper(angularHelper, serverValidationManager, notificationsService
 
             if(firstInvalidNgForm.length !== 0) {
                 var focusableFields = [...firstInvalidNgForm.find("umb-range-slider .noUi-handle,input,textarea,select,button")];
-                if(focusableFields.length !== 0) { 
+                if(focusableFields.length !== 0) {
                     var firstErrorEl = focusableFields.find(el => el.type !== "hidden" && el.hasAttribute("readonly") === false);
-                    if(firstErrorEl.length !== 0) {
+                    if(firstErrorEl !== undefined) {
```

Comment:
@nielslyngsoe why not just the following, which would also check for `null`:

```
if(firstErrorEl) {
    firstErrorEl.focus();
}
```

Reply:
Well, that's also a fine solution. I just wrote this cause based on what the Mozilla docs says:
"If no values satisfies the testing function, undefined is returned."
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
Personally, I'd only check length if I need the exact value - if I only need to know that the array has items, `if (arr.length)` is fine. If I'm checking a variable exists, I'd do as @bjarnef suggests.

Reply:
Alternatively I like to use `if(x != null)` which checks for both `null` and `undefined`. 
Pretty much the only case where `!=` is fine (vs. `!==`).

Reply:
I'd avoid abstract equality (==) without type safety. Intellisense should warn me if I'm comparing different types so that I only do it knowingly and without relying on coercion. 

If I'm comparing two variables, they are different if they are not the same type. If I need to compare a string and a number, I'll parse/toString one or the other first. 

Personal preference.